crates/primitives/Cargo.toml

```diff
@@ -66,6 +66,7 @@ arbitrary = ["std", "alloy-primitives/arbitrary", "bitflags/arbitrary"]
 asm-keccak = ["alloy-primitives/asm-keccak"]
 portable = ["c-kzg?/portable"]
 
+bsc = []
```

Comment:
does this make sense if we write a empty packages here?

Reply:
Yes.